omero/sysadmins/unix/index.txt

```diff
@@ -14,9 +14,19 @@ additional pages listed below.
 
     server-installation
     server-centos6
+    server-centos6-ice36
     server-centos7
     server-linux-walkthrough
     server-install-homebrew
     server-binary-repository
     server-postgresql
     install-web
+
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    walkthrough/requirements
+    walkthrough/requirements_centos6
```

Comment:
walkthrough/walkthrough_centos6.txt does not need to be added to the index since it is not present